platform/android/java/editor/src/meta/java/org/godotengine/editor/GodotEditor.kt

```diff
@@ -91,4 +91,12 @@ open class GodotEditor : BaseGodotEditor() {
 			else -> super.getEditorWindowInfoForInstanceId(instanceId)
 		}
 	}
+
+	override fun supportsFeature(featureTag: String): Boolean {
+		if (featureTag == "xr_editor") {
+			return true
```

Comment:
Shouldn't this be calling `isNativeXRDevice()`?

```suggestion
			return isNativeXRDevice()
```

Reply:
That's a good idea! If we use that method, we can move this logic to `BaseGodotEditor` to make it platform agnostic.